apps/admin/src/App.js

```diff
@@ -13,7 +13,7 @@ import productionManager from "./request/production-manager";
 const App = () => (
     <Admin authProvider={authProvider} dataProvider={dataProvider}>
         {permissions => [
-            <ResourceWithPermissions
+            permissions['dispatcher-requests'] && permissions['dispatcher-requests'].enabled && <ResourceWithPermissions
```

Comment:
This is supposed to work without the condition

Reply:
Yop, but it do not ^^

Reply:
Meh !